modules/claims_api/app/controllers/concerns/claims_api/document_validations.rb

```diff
@@ -32,10 +32,12 @@ def document_content_type_errors
         end
       end
 
+      # rubocop:disable Layout/LineLength
       def pdf?(document)
         extension = document.original_filename.split('.').last
-        ['application/pdf', 'text/plain'].include?(document.content_type) && extension.downcase == 'pdf'
+        ['application/pdf', 'text/plain', 'application/octet-stream'].include?(document.content_type) && extension.downcase == 'pdf'
```

Comment:
reason for this addition is a ruby quirk. basically if `Content-Type` is unspecified, it defaults to `application/octet-stream`. I think if the extension is still `pdf` this is a safe thing to allow